src/bloggy_frontend/src/pages/Posts.jsx

```diff
@@ -15,7 +17,7 @@ const Posts = () => {
     error,
     isLoading,
   } = useQuery({
-    queryKey: ['posts'],
+    queryKey: [QUERY_KEYS.POSTS],
     queryFn: fetchPosts,
   })
```

Comment:
move this also to the posts.js file

Reply:
didn't understand this, could you explain more?